ubi.repo

```diff
@@ -0,0 +1,62 @@
+[ubi-9-for-$basearch-baseos-rpms]
+name = Red Hat Universal Base Image 9 (RPMs) - BaseOS
+baseurl = https://cdn-ubi.redhat.com/content/public/ubi/dist/ubi9/9/$basearch/baseos/os
+enabled = 1
+gpgkey = file:///etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release
+gpgcheck = 1
```

Comment:
I believe the sslclient key and certification are required for future generation of the lockfile locally. the k5x ci might not detect this at the moment but if not mistaken it is needed by the mint maker. please do verify this matter.